### PR TITLE
[zh-cn]sync labels-annotations-taints/index.md

### PR DESCRIPTION
**Pull request by asa3311**
sync labels-annotations-taints/index.md

### CODE REVIEW

File: content/zh-cn/docs/reference/labels-annotations-taints/_index.md [hunk 1]
```diff
@@ -208,6 +208,105 @@ If this annotation is not set then the cluster autoscaler follows its Pod-level
...
+
+Used on: All objects
+
+This annotation is used in manifests to mark an object as local configuration that should not be submitted to the Kubernetes API.
+A value of "true" for this annotation declares that the object is only consumed by client-side tooling and should not be submitted to the API server.
+A value of "false" can be used to declare that the object should be submitted to the API server even when it would otherwise be assumed to be local.
+This annotation is part of the Kubernetes Resource Model (KRM) Functions Specification, which is used by Kustomize and similar third-party tools. For example, Kustomize removes objects with this annotation from its final build output.
+-->
+
+### config.kubernetes.io/local-config {#config-local-config}
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/labels-annotations-taints/_index.md:224]: ref: https://kubernetes.io/docs/reference/labels-annotations-taints/#config-kubernetes-io-local-config
```suggestion
### config.kubernetes.io/local-config {#config-kubernetes-io-local-config}
```

File: content/zh-cn/docs/reference/labels-annotations-taints/_index.md [hunk 2]
```diff
@@ -208,6 +208,105 @@ If this annotation is not set then the cluster autoscaler follows its Pod-level
...
+
+例子：`config.kubernetes.io/local-config: "true"`
+
+用于：所有对象
+
+该注解用于标记 Kubernetes 配置文件中的对象，表示这些对象是本地配置，不应提交到 Kubernetes API。
+对于这个注解，当值为 "true"时，表示该对象仅被客户端工具使用，不应提交到 API 服务器。
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/labels-annotations-taints/_index.md:231]:
```suggestion
对于这个注解，当值为 “true” 时，表示该对象仅被客户端工具使用，不应提交到 API 服务器。
```

File: content/zh-cn/docs/reference/labels-annotations-taints/_index.md [hunk 3]
```diff
@@ -208,6 +208,105 @@ If this annotation is not set then the cluster autoscaler follows its Pod-level
...
+This annotation is part of the Kubernetes Resource Model (KRM) Functions Specification, which is used by Kustomize and similar third-party tools. For example, Kustomize removes objects with this annotation from its final build output.
+-->
+
+### config.kubernetes.io/local-config {#config-local-config}
+
+例子：`config.kubernetes.io/local-config: "true"`
+
+用于：所有对象
+
+该注解用于标记 Kubernetes 配置文件中的对象，表示这些对象是本地配置，不应提交到 Kubernetes API。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/labels-annotations-taints/_index.md:230]:
```suggestion
该注解用于清单中的对象，表示某对象是本地配置，不应提交到 Kubernetes API。
```

File: content/zh-cn/docs/reference/labels-annotations-taints/_index.md (hunk 2 above)
Comment (tengqm) [inline content/zh-cn/docs/reference/labels-annotations-taints/_index.md:231]:
```suggestion
对于这个注解，当值为 "true" 时，表示该对象仅被客户端工具使用，不应提交到 API 服务器。
```

File: content/zh-cn/docs/reference/labels-annotations-taints/_index.md [hunk 4]
```diff
@@ -208,6 +208,105 @@ If this annotation is not set then the cluster autoscaler follows its Pod-level
...
+<!--
+### internal.config.kubernetes.io/* (reserved prefix) {#internal.config.kubernetes.io-reserved-wildcard}
+
+Used on: All objects
+
+This prefix is reserved for internal use by tools that act as orchestrators in accordance with the Kubernetes Resource Model (KRM) Functions Specification. Annotations with this prefix are internal to the orchestration process and are not persisted to the manifests on the filesystem. In other words, the orchestrator tool should set these annotations when reading files from the local filesystem and remove them when writing the output of functions back to the filesystem.
+A KRM function **must not** modify annotations with this prefix, unless otherwise specified for a given annotation. This enables orchestrator tools to add additional internal annotations, without requiring changes to existing functions.
+-->
+
+### internal.config.kubernetes.io/* (reserved prefix) {#internal.config.kubernetes.io-reserved-wildcard}
```
Comment (tengqm) [inline content/zh-cn/docs/reference/labels-annotations-taints/_index.md:246]:
```suggestion
### internal.config.kubernetes.io/* （保留的前缀） {#internal.config.kubernetes.io-reserved-wildcard}
```

File: content/zh-cn/docs/reference/labels-annotations-taints/_index.md [hunk 5]
```diff
@@ -208,6 +208,105 @@ If this annotation is not set then the cluster autoscaler follows its Pod-level
...
+
+用于：所有对象
+
+该前缀被保留，供按照 Kubernetes 资源模型 (KRM) 函数规范的编排工具内部使用。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/labels-annotations-taints/_index.md:250]:
```suggestion
该前缀被保留，供遵从 Kubernetes 资源模型 (KRM) 函数规范的编排工具内部使用。
```

File: content/zh-cn/docs/reference/labels-annotations-taints/_index.md [hunk 6]
```diff
@@ -208,6 +208,105 @@ If this annotation is not set then the cluster autoscaler follows its Pod-level
...
+
+### config.kubernetes.io/local-config {#config-local-config}
+
+例子：`config.kubernetes.io/local-config: "true"`
+
+用于：所有对象
+
+该注解用于标记 Kubernetes 配置文件中的对象，表示这些对象是本地配置，不应提交到 Kubernetes API。
+对于这个注解，当值为 "true"时，表示该对象仅被客户端工具使用，不应提交到 API 服务器。
+当值为 "false"，可以用来声明该对象应提交到 API 服务器，即使它是本地对象。
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/labels-annotations-taints/_index.md:232]:
```suggestion
当值为 “false” 时，可用于声明该对象应提交到 API 服务器，即使它是本地对象。
```

File: content/zh-cn/docs/reference/labels-annotations-taints/_index.md [hunk 7]
```diff
@@ -208,6 +208,105 @@ If this annotation is not set then the cluster autoscaler follows its Pod-level
...
+A KRM Function **should not** modify this annotation on input objects unless it is modifying the referenced files. A KRM Function **may** include this annotation on objects it generates.
+-->
+
+### internal.config.kubernetes.io/path {#internal-config-path}
+
+例子：`internal.config.kubernetes.io/path: "relative/file/path.yaml"`
+
+用于：所有对象
+
+此注解记录了加载对象清单文件的（斜杠分隔、与操作系统无关）相对路径。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/labels-annotations-taints/_index.md:275]:
```suggestion
此注解记录了加载对象清单文件的（斜线分隔、与操作系统无关）相对路径。
```

File: content/zh-cn/docs/reference/labels-annotations-taints/_index.md (hunk 5 above)
Comment (howieyuen) [inline content/zh-cn/docs/reference/labels-annotations-taints/_index.md:250]:
```suggestion
该前缀为 Kubernetes 资源模型（KRM）函数规范所规定的编排工具内部使用所保留。
```

File: content/zh-cn/docs/reference/labels-annotations-taints/_index.md [hunk 8]
```diff
@@ -208,6 +208,105 @@ If this annotation is not set then the cluster autoscaler follows its Pod-level
...
+Example: `internal.config.kubernetes.io/path: "relative/file/path.yaml"`
+
+Used on: All objects
+
+This annotation records the slash-delimited, OS-agnostic, relative path to the manifest file the object was loaded from. The path is relative to a fixed location on the filesystem, determined by the orchestrator tool.
+This annotation is part of the Kubernetes Resource Model (KRM) Functions Specification, which is used by Kustomize and similar third-party tools.
+A KRM Function **should not** modify this annotation on input objects unless it is modifying the referenced files. A KRM Function **may** include this annotation on objects it generates.
+-->
+
+### internal.config.kubernetes.io/path {#internal-config-path}
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/labels-annotations-taints/_index.md:269]:
```suggestion
### internal.config.kubernetes.io/path {#internal-config-kubernetes-io-path}
```

File: content/zh-cn/docs/reference/labels-annotations-taints/_index.md (hunk 7 above)
Comment (howieyuen) [inline content/zh-cn/docs/reference/labels-annotations-taints/_index.md:275]:
```suggestion
此注解记录了加载对象的清单文件的相对路径（斜杠分隔、与操作系统无关）。
```

File: content/zh-cn/docs/reference/labels-annotations-taints/_index.md [hunk 9]
```diff
@@ -208,6 +208,105 @@ If this annotation is not set then the cluster autoscaler follows its Pod-level
...
+Example: `internal.config.kubernetes.io/index: "2"`
+
+Used on: All objects
+
+This annotation records the zero-indexed position of the YAML document that contains the object within the manifest file the object was loaded from. Note that YAML documents are separated by three dashes (`---`) and can each contain one object. When this annotation is not specified, a value of 0 is implied.
+This annotation is part of the Kubernetes Resource Model (KRM) Functions Specification, which is used by Kustomize and similar third-party tools.
+A KRM Function **should not** modify this annotation on input objects unless it is modifying the referenced files. A KRM Function **may** include this annotation on objects it generates.
+-->
+
+### internal.config.kubernetes.io/index {#internal-config-index}
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/labels-annotations-taints/_index.md:295]:
```suggestion
### internal.config.kubernetes.io/index {#internal-config-kubernetes-io-index}
```

**Review by howieyuen — CHANGES_REQUESTED** [review]

**Review by howieyuen — COMMENTED** [review]
/lgtm

**Review by ydFu — APPROVED** [review]
/lgtm

**Pull request merged**